test/Microsoft.AspNet.Razor.Runtime.Test/TagHelpers/TagHelperOutputTest.cs

```diff
@@ -147,6 +173,39 @@ public void GenerateEndTag_ReturnsNothingIfWhitespaceTagName()
         }
 
         [Fact]
+        public void GeneratePreContent_ReturnsPreContent()
+        {
+            // Arrange
+            var tagHelperOutput = new TagHelperOutput("p");
+
+            tagHelperOutput.PreContent = "Hello World";
+
+            // Act
+            var output = tagHelperOutput.GeneratePreContent();
+
+            // Assert
+            Assert.Equal("Hello World", output);
+        }
+
+        [Fact]
+        public void GeneratePreContent_ReturnsNothingIfSelfClosing()
+        {
+            // Arrange
+            var tagHelperOutput = new TagHelperOutput("p")
+            {
+                SelfClosing = true
+            };
+
+            tagHelperOutput.PreContent = "Hello World";
```

Comment:
nit: particularly weird to have both full assignment and init syntax.  add `PreContent` to the existing initialization